tools/osbuilder/node-builder/azure-linux/package_build.sh

```diff
@@ -41,29 +41,27 @@ fi
 
 agent_make_flags="LIBC=gnu OPENSSL_NO_VENDOR=Y DESTDIR=${AGENT_INSTALL_DIR} BUILD_TYPE=${AGENT_BUILD_TYPE}"
 
-if [ "${CONF_PODS}" == "yes" ]; then
-	agent_make_flags+=" AGENT_POLICY=yes"
-fi
+#if [ "${CONF_PODS}" == "yes" ]; then
```

Comment:
Let's talk - we should factor our tardev into its own targets etc.

Reply:
Separated tardev related builds into a separate script.